apps/web/src/components/tx/security/tenderly/index.tsx

```diff
@@ -136,7 +137,7 @@ const TxSimulationBlock = ({
               }}
             />
           ) : isFinished ? (
-            !isSuccess || isError || isCallTraceError ? (
+            !isSuccess || isError ? (
```

Comment:
Now that we also have a separate case for isCallTraceError I would prefer to refactor the whole component rendering logic instead of doing more nested ternaries.